templates/web/base/maps/noscript_map_wms.html

```diff
@@ -0,0 +1,32 @@
+<div class="noscript square-map__outer">
+    <div class="square-map__inner">
+        <div id="[% nsm_prefix %]drag">
+            [%- FOR row IN map.tiles -%]
+                [%- FOR tile IN row -%]
+                    [%- top_px = tile.row_offset * map.tile_size -%]
+                    [%- left_px = tile.col_offset * map.tile_size %]
+                    <[% map.img_type %]
+                        class="square-map__tile"
+                        alt="[% tile.alt %]"
+                        id="[% nsm_prefix %]t[% tile.dotted_id %]"
+                        name="tile_[% tile.dotted_id %]"
+                        src="[% tile.src %]"
+                        style="width: [% map.tile_size %]px; top: [% top_px %]px; left: [% left_px %]px;">
+                [%- END -%]
+            [% END %]
+        </div>
+        <div id="[% nsm_prefix %]pins">[% FOR pin IN map.pins %][% INCLUDE pin %][% END %]</div>
+        [% INCLUDE 'maps/_compass.html' %]
+    </div>
+</div>
+
+[% BLOCK pin %]
+[%
+    SET pin_top = pin.py ;
```

Comment:
I don't think you need any of this - if you only include maps/pin.html, is it not okay? That shifts it 24/64, just like this is doing.